html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-forms</title>
+</head>
+
+<body>
+  <h1><strong>Coding Form!</strong></h1>
+  <form>
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <input id="email" type="text" name="email" size="30" maxlength="100">
+    <div>Why is coding so cool?</div>
```

Comment:
You should never use `div` elements as text elements, this should be a `p` element.